jobs/JGDAS_ATMOS_GEMPAK

```diff
@@ -40,26 +30,22 @@ for grid in 0p25 1p00; do
   fi
 done
 
-# TODO: These actions belong in an ex-script not a j-job
-if [[ -f poescript ]]; then
-   rm -f poescript
+########################################################
+# Execute the script for one degree grib
+########################################################
+fhmin=0
+fhmax=9
+if (( fcsthrs >= fhmin && fcsthrs <= fhmax )); then
+  if ((fcsthrs % 3 == 0)); then
+    "${SCRgfs}/exgdas_atmos_nawips.sh" "1p00" "${fcsthrs}" "GDAS_GEMPAK" "${COM_ATMOS_GEMPAK_1p00}"
+
+########################################################
+# Execute the script for quater-degree grib
+########################################################
+    "${SCRgfs}/exgdas_atmos_nawips.sh" "0p25" "${fcsthrs}" "GDAS_GEMPAK" "${COM_ATMOS_GEMPAK_0p25}"
+  fi
 fi
```

Comment:
The `if`s shouldn't be necessary anymore. If we need to modify the hours that are run in the metatask, we can do that.

Reply:
`if` statements are removed.